.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Create a report to help us improve something that is not working correctly
+title: 'Bug - :'
```

Nit due to the checkbox about format
```suggestion
title: 'Bug - '
```